notebooks/test_2.ipynb

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
<p>This looks nice but you can only comment on a single markdown cell at a time which may be problematic, Though we may just encourage the use of more smaller markdown cells and less singular large ones.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/moaatt2/reviewnb-test/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>This is a neat idea and should be taken into consideration.</p>

Reply:
You can add to the conversation here but the part of the code it shows on GitHub isn't great.